src/main/java/com/flightstats/hub/webhook/WebhookCoordinator.java

```diff
@@ -7,6 +7,7 @@
 import com.flightstats.hub.config.properties.WebhookProperties;
 import com.flightstats.hub.dao.Dao;
 import com.flightstats.hub.model.ContentPath;
+import com.flightstats.hub.webhook.strategy.WebhookStrategy;
 import com.google.common.annotations.VisibleForTesting;
```

Comment:
this and the next two changed files look a little odd because there is a new import, but not a change as to the import's use in the file, just checking that these are intentional

Reply:
Hopefully I understand your question correctly: When I moved the webhook strategy stuff into its own package, it required me to import them in all the classes that used it and with which it used to live (so anything in just "com.flightstats.hub.webhook"). That's why there's no code changed: the file(s) just moved.